packages/react-import-remote/src/ImportRemote.tsx

```diff
@@ -5,6 +5,10 @@ import {
   WindowWithRequestIdleCallback,
   DeferTiming,
 } from '@shopify/async';
+import {
```

Comment:
Isn't this prettymuch unshakeable? :/  Would it make more sense for `react-intersection-observer` to provide an intersectable extension to import remote?

Reply:
It is not tree shakeable, but the module is very small. I don't think we can cleanly expose a way to inject stuff in that doesn't rely on shaky type checking and module-scoped variables. I know this isn't ideal for consumers that don't need everything, but web is probably the only real consumer of this right now, and we'll deal with better tree shaking for all of these later.